guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+entityDefinitions:
+
+
+
+  offenders:
+    fqn: "general.person"
+    entitySetName: "GuilfordOffenders"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX"]
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "COMMITTED LAST NAME"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "COMMITTED FIRST NAME"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "COMMITTED MIDDLE NAME"
+          pattern: "None"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "COMMITTED MIDDLE NAME"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      nc.PersonSuffix:
+        type: "nc.PersonSuffix"
+        column: "COMMITTED NAME SUFFIX"
+      ol.datasource: # value transform - write nc dps
+        type: "ol.datasource"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "NC DPS"
+    name: "offenders" 
+
+  sentences:
+    fqn: "ol.sentence"
+    entitySetName: "GuilfordSentences_DPS"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "OFFENDER NC DOC ID NUMBER"
+    name: "sentences"
+
+  sentenceterms:
+    fqn: "ol.sentenceterm"
+    entitySetName: "GuilfordSentenceTerms"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "EARLIEST SENTENCE EFFECTIVE DT"]
+      general.fullname: 
+        type: "general.fullname"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TOTAL SENTENCE LENGTH"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TOTAL SENTENCE LENGTH"
+          - !<transforms.ReplaceTransform>
+            target: ["9999999", "9999998"]
+            goal: ["Death Sentence", "Life Sentence"]
+            valueElse: null
+          transformsIfTrue:
```

Comment:
You don't need to specifiy`transformsIfTrue` if you don't wanna do anything.

Reply:
Good to know, changed this in all instances with transformsIfTrue -> ""